google/cloud/bigquery/client.py

```diff
@@ -656,10 +698,13 @@ def test_iam_permissions(
         body = {"permissions": permissions}
 
         path = "{}:testIamPermissions".format(table.path)
-
-        response = self._call_api(
-            retry, method="POST", path=path, data=body, timeout=timeout,
-        )
+        span_attributes = {"path": path, "body": body}
+        with create_span(
+            name="BigQuery.testIamPermissions", attributes=span_attributes, client=self
```

Comment:
```suggestion
            name="BigQuery.testIAMPermissions", attributes=span_attributes, client=self
```
(for consistency with `getIAMPolicy` and `setIAMPolicy`)